packages/cli-plugin/.eslintrc.cjs

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  root: true,
+  extends: ['curvenote'],
+  rules: {
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: ['**/*.spec.ts'],
+      },
+    ],
+  },
```

Comment:
This only works if vitest is in the devDeps, and I think that version mismatches are worse. My advice is to put:

```
eslint \"src/**/!(*.spec).ts\" -c ./.eslintrc.cjs
```

In the lint function, and ignore them from tests/CI.

Then if the red line still is annoying:

```
/* eslint-disable import/no-extraneous-dependencies */
```